ext/drmgr/drmgr.h

```diff
@@ -1221,8 +1221,9 @@ DR_EXPORT
  * behaves just like DR's signal event dr_register_signal_event().
  * \return whether successful.
  */
-bool drmgr_register_signal_event(dr_signal_action_t (*func)(void *drcontext,
-                                                            dr_siginfo_t *siginfo));
+bool
```

Comment:
The same clang-format off needs to be here and below to satisfy both vera++ and clang-format in the presence of the clang-format bug.

Reply:
Done